app/views/mechanics/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1> <%= @mechanic.name %></h1>
+<p>Years of Experience: <%= @mechanic.years_experience %></p>
+
+<h2>Rides working on:</h2>
+<ul>
+    <% @mechanic.rides.each do |ride| %>
+        <li><%= ride.name %></li>
+    <% end %>
+</ul>
+
+<h2>Add a Ride:</h2>
+<%= form_with url: add_ride_mechanic_path(@mechanic), method: :post do %>
+    <label for="ride_id">Ride Id:</label>
+    <%= text_field_tag :ride_id %>
+    <%= submit_tag "Submit" %>
```

Comment:
This form syntax will work, but it's not the correct syntax. Review the lesson on forms to see the proper syntax for building a form.